SDL/Quintuplet.cu

```diff
@@ -433,51 +433,52 @@ __device__ bool SDL::runQuintupletDefaultAlgo(struct SDL::modules& modulesInGPU,
+        sqrtf(x2*x2 + y2*y2),                                                        // inner T3 anchor hit 2 r (t3_2_r)
+        float(modulesInGPU.layers[lowerModuleIndex2] + 6*is_endcap2),                // inner T3 anchor hit 2 layer (t3_2_layer)
+        mdsInGPU.anchorEta[thirdMDIndex],                                            // inner T3 anchor hit 3 eta (t3_4_eta)
+        mdsInGPU.anchorPhi[thirdMDIndex],                                            // inner T3 anchor hit 3 phi (t3_4_phi)
+        mdsInGPU.anchorZ[thirdMDIndex],                                              // inner T3 anchor hit 3 z (t3_4_z)
+        sqrtf(x3*x3 + y3*y3),                                                        // inner T3 anchor hit 3 r (t3_4_r)
+        float(modulesInGPU.layers[lowerModuleIndex3] + 6*is_endcap3),                // inner T3 anchor hit 3 layer (t3_4_layer)
+        log10(2*SDL::k2Rinv1GeVf*outerRadius),                                       // outer T3 pT (t3_pt)
+        mdsInGPU.anchorEta[thirdMDIndex],                                            // outer T3 anchor hit 4 eta (t3_0_eta)
+        mdsInGPU.anchorPhi[thirdMDIndex],                                            // outer T3 anchor hit 4 phi (t3_0_phi)
+        mdsInGPU.anchorZ[thirdMDIndex],                                              // outer T3 anchor hit 3 eta (t3_0_z)
+        sqrtf(x3*x3 + y3*y3),                                                        // outer T3 anchor hit 3 r (t3_0_r)
+        float(modulesInGPU.layers[lowerModuleIndex3] + 6*is_endcap3),                // outer T3 anchor hit 3 layer (t3_0_layer)
+        mdsInGPU.anchorEta[fourthMDIndex],                                           // outer T3 anchor hit 4 eta (t3_2_eta)
+        mdsInGPU.anchorPhi[fourthMDIndex],                                           // outer T3 anchor hit 4 phi (t3_2_phi)
+        mdsInGPU.anchorZ[fourthMDIndex],                                             // outer T3 anchor hit 4 z (t3_2_z)
+        sqrtf(x4*x4 + y4*y4),                                                        // outer T3 anchor hit 4 r (t3_2_r)
+        float(modulesInGPU.layers[lowerModuleIndex4] + 6*is_endcap4),                // outer T3 anchor hit 4 layer (t3_2_layer)
+        mdsInGPU.anchorEta[fifthMDIndex],                                            // outer T3 anchor hit 5 eta (t3_4_eta)
+        mdsInGPU.anchorPhi[fifthMDIndex],                                            // outer T3 anchor hit 5 phi (t3_4_phi)
+        mdsInGPU.anchorZ[fifthMDIndex],                                              // outer T3 anchor hit 5 z (t3_4_z)
+        sqrtf(x5*x5 + y5*y5),                                                        // outer T3 anchor hit 5 r (t3_4_r)
+        float(modulesInGPU.layers[lowerModuleIndex5] + 6*is_endcap5),                // outer T3 anchor hit 5 layer (t3_4_layer)
+        log10((innerRadius + outerRadius)*SDL::magnetic_field*1.602f/(2*100*5.39f)), // T5 pT (t5_pt)
```

Comment:
I forgot to mention it in the commit message, but I have replaced `3.8f` with `SDL::magnetic_field`